src/Faction/ui/FactionsRoot.tsx

```diff
@@ -51,6 +58,28 @@ export function FactionsRoot(props: IProps): React.ReactElement {
     setRerender((x) => !x);
   }
 
+  const getAugsLeft = (faction: Faction, player: IPlayer): number => {
+    const isPlayersGang = player.inGang() && player.getGangName() === faction.name;
```

Comment:
faction.name might cause the recovery screen if they have it open and augment, as Factions[faction] is a global thats not super safe. though its pretty easy to verify.

just run a script that timeouts out for 15 then augments with this page open

might be worth putting Factions[faction] in a variable at the top and doing an early out to avoid the issue ( thought i did this but maybe it got rolled back?)

Reply:
tested, can verify that there are no issues